src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
There are no reasons to extend ColorSupplier better to create an instance of CollorSupplier in the Lottery and use it.